client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -104,7 +104,7 @@ class EditorRevisionsList extends PureComponent {
 	};
 
 	render() {
-		const { revisions, selectedRevisionId, siteId } = this.props;
+		const { postId, revisions, selectedRevisionId, siteId } = this.props;
```

Comment:
When the content in the first change is too long, it's not auto selecting the first item in the list.

Reply:
Yep, can confirm it's working for me now!